cranelift/codegen/src/isa/x64/inst/args.rs

```diff
@@ -209,6 +209,30 @@ impl fmt::Debug for RMI_R_Op {
 /// These indicate ways of extending (widening) a value, using the Intel naming:
 /// B(yte) = u8, W(ord) = u16, L(ong)word = u32, Q(uad)word = u64
 #[derive(Clone, PartialEq)]
+// Some basic ALU operations.
+pub enum FP_RM_R_Op {
+    Addss,
+    Subss,
+}
+
+impl FP_RM_R_Op {
```

Comment:
We should really rename both this and the existing `RM_R_Op` to something more sensible.  For `RM_R_Op` I'm not sure what.  For this, maybe `SSE2_BinOp` would be good.  Basically all two-operand SSE2 instructions (the R/R and R/M) versions can be represented this way, so it's nice and compact.

Reply:
I guess `RM_R_Op` should then be changed to `INT_RM_R_Op`.